CMakeLists.txt

```diff
@@ -214,15 +218,18 @@ add_subdirectory(models)
 add_subdirectory(Source)
 add_subdirectory(docs)
 
+
 if(DPSIM_BUILD_EXAMPLES)
 	add_subdirectory(Examples)
 endif(DPSIM_BUILD_EXAMPLES)
 
 if(WITH_PYBIND)
-	set(PYBIND11_CPP_STANDARD -std=c++11)
+	set(DPSIMPY_INCLUDE_DIRS
+		${CMAKE_CURRENT_BINARY_DIR}/Include/dpsim/pybind
```

Comment:
@JTS22 just for curiosity... why have you added the binary directory here?

Reply:
I probably just copied it from the other include directives. It should not be necessary, so i removed the binary dir include in 786a0ab2985ce510ac46da2823dd4a1f94899d07